cmd/generate/config/rules/grafana.go

```diff
@@ -0,0 +1,65 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func GrafanaApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Grafana api key (or Grafana cloud api key)",
+		RuleID:      "grafana-api-key",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`eyJrIjoi[A-Za-z0-9]{70,}={0,2}`),
+		Keywords:    []string{"eyJrIjoi"},
```

Comment:
@jmatosgrafana anyway we could put an upper bound on the `{70,}` quantifier? Not a huge deal but typically for all the rules in the default gitleaks configuration everything is bounded.

Reply:
@zricethezav Thank you for the quick feedback. Here is the answer we provided to GitHub secret scanning team:
_it's a bit hard to predict the largest size but I'd reasonably expect a GRAFANA_API_KEY token not to exceed ~400 characters_

Would 400 be a reasonable upper bound for you ?